### DISCUSSION TITLE
FreeRDP for RHEL V7.9

### DISCUSSION THREAD

**Discussion by roewan** [FreeRDP/FreeRDP#8182] [Q&A]
Is it possible to install FreeRDP in RHEL V7.9?
I am trying to install Apache Guacamole in the server and one of the packages required is freerdp-devel.
Any help is greatly appreciated. Thank you.

**Comment by akallabeth** [comment]
Don´t know about binaries, but to build:

* there is a small `spec` file to build the nightly in the repo
* You can modify existing rhel `spec` files to use the version you want
